xml/System.IO/FileSystemInfo.xml

```diff
@@ -189,7 +189,9 @@
   
 -   <xref:System.IO.DirectoryInfo.EnumerateFileSystemInfos%2A>  
   
- To get the latest value, call the <xref:System.IO.FileSystemInfo.Refresh%2A> method.  
+ The value is cached when it or length or time properties are retrieved. To get the latest value, call the <xref:System.IO.FileSystemInfo.Refresh%2A> method.
```

Comment:
The value may be cached when either the value itself or other FileSystemInfo properties are accessed.

Is that better?